app/Http/Controllers/TarnibController.php

```diff
@@ -139,444 +188,395 @@ function joinRoom(Request $request){
-        ]);
-
-        $round = DB::table('rounds')->where('id', $room->first()->round_id)->first();
-        $player_cards = 'player_'.$request->seat.'_cards';
-        
-        broadcast(new \App\Events\TarnibChooseSeatEvent($request->seat, $request->player_name, $request->room_id))->toOthers();
-
-        return response()->json([
-            'player_cards' => json_decode($round->$player_cards)
-        ]);
-    }
-
-    function sortCards($cards){
-        $spades = [];
-        $hearts = [];
-        $clubs = [];
-        $diamonds = [];
-
-        foreach ($cards as $card) {
-            if(str_contains($card, 'spades')){
-                $spades[] = $card;
-            }elseif (str_contains($card, 'hearts')) {
-                $hearts[] = $card;
-            }elseif (str_contains($card, 'clubs')) {
-                $clubs[] = $card;
-            }else{
-                $diamonds[] = $card;
+        function sortCards($cards){
+            $spades = [];
+            $hearts = [];
+            $clubs = [];
+            $diamonds = [];
+
+            foreach ($cards as $card) {
+                if(str_contains($card, 'spades')){
+                    $spades[] = $card;
+                }elseif (str_contains($card, 'hearts')) {
+                    $hearts[] = $card;
+                }elseif (str_contains($card, 'clubs')) {
+                    $clubs[] = $card;
+                }else{
+                    $diamonds[] = $card;
+                }
             }
-        }
 
-        $spades = $this->sortSuit($spades, 'spades');
-        $hearts = $this->sortSuit($hearts, 'hearts');
-        $clubs = $this->sortSuit($clubs, 'clubs');
-        $diamonds = $this->sortSuit($diamonds, 'diamonds');
+            $spades = $this->sortSuit($spades, 'spades');
+            $hearts = $this->sortSuit($hearts, 'hearts');
+            $clubs = $this->sortSuit($clubs, 'clubs');
+            $diamonds = $this->sortSuit($diamonds, 'diamonds');
 
-        $res = array_merge($hearts,$spades);
-        $res = array_merge($res, $diamonds);
-        $res = array_merge($res, $clubs);
-        return $res;
-    }
+            $res = array_merge($hearts,$spades);
+            $res = array_merge($res, $diamonds);
+            $res = array_merge($res, $clubs);
+            return $res;
+        }
 
-    function sortSuit($cards, $suit){
-        $suffix = "_of_$suit";
-
-        
-        for($i = 0; $i < count($cards); $i++){
-            for($j = 0; $j < count($cards)-1; $j++){
-                $val1 = str_replace($suffix, "",$cards[$j]);
-                $val2 = str_replace($suffix, "",$cards[$j+1]);
-                if ($val2 > $val1) {
-                    $temp = $cards[$j];
-                    $cards[$j] = $cards[$j+1];
-                    $cards[$j+1] = $temp;
+        function sortSuit($cards, $suit){
+            $suffix = "_of_$suit";
+
+            
+            for($i = 0; $i < count($cards); $i++){
```

Comment:
Also can you add comments explaining the sorting ? 

nits for better redability: `count(cards)` can be a reused variable, also you  should ideally use a built in sorting function (not a concern here)